src/Blockfrost.ConsoleTool/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Blockfrost.ConsoleTool.Commands;
+
+namespace Blockfrost.ConsoleTool
+{
+    /// <summary>
+    /// Based on https://github.com/CardanoSharp/cscli
+    /// </summary>
+    class Program
+    {
+        static async Task<int> Main(string[] args)
```

Comment:
Oh and I also added a console tool based on the cscli project.
I think we can do some great things using the commands they are building!